### PR TITLE
Qmaps 2158 help center links

### PR DESCRIPTION
**Pull request by xem**
## Description
- Provide up-to-date links for the no geolocation & no result found help pages in 4 languages
- Display buttons in geolocation modals and no-result suggest that redirect to these help pages in a new tab

## Screenshots
image
image

### CODE REVIEW

File: src/modals/GeolocationModal.jsx [hunk 1]
```diff
@@ -51,21 +51,27 @@ const GeolocationModal = ({ status, onClose, onAccept }) => {
...
       className: 'modal__maps__denied',
     },
     NOT_ACTIVATED: {
       title: _('Houston,<br/> we have a (geolocation) problem&nbsp;🛰️', 'geolocation'),
       text: _(
         "We can't access your position.<br/> Please check that your geolocation services are enabled."
       ),
-      button: _('Ok', 'geolocation'),
+      link: {
+        label: _('How to access the geolocation services?', 'geolocation'),
```
Comment (fatal69100) [inline src/modals/GeolocationModal.jsx:66]: Should we add a space before the `?` :  `How to access the geolocation services ?`

File: config/constants.yml [hunk 2]
```diff
@@ -39,9 +39,19 @@ localizedUrls:
     aboutPrivacy: https://about.qwant.com/legal/privacy
     aboutMapsToS: https://about.qwant.com/legal/terms-of-service/qwant-maps/
     contributing: https://github.com/Qwant/qwantmaps/blob/master/contributing.md
+    helpGeolocation: https://help.qwant.com/en/docs/maps/activation-of-geolocation-after-denied-access/
```
Comment (fatal69100) [inline config/constants.yml:42]: It does not depend on this PR, but images contained into these links are missing for me (on Firefox) :sweat_smile:

File: src/panel/NoResultMessage.jsx [hunk 3]
```diff
@@ -16,6 +20,15 @@ const NoResultMessage = () => {
           'suggest'
         )}
       </p>
+      <p className="u-center">
+        <Button
+          href={helpCenterLink}
+          variant="tertiary"
+          onMouseDown={() => window.open(helpCenterLink)}
+        >
+          <IconExternalLink /> {_('Add a missing place on the map')}
```
Comment (fatal69100) [inline src/panel/NoResultMessage.jsx:29]: Should the text only be in english ?

File: src/modals/GeolocationModal.jsx (hunk 1 above)
Comment (remi-dupre) [reply src/modals/GeolocationModal.jsx:66]: There isn't any space before punctuation in English, apparently this is a typically French mistake: https://ell.stackexchange.com/a/4870 :laughing:

File: config/constants.yml (hunk 2 above)
Comment (remi-dupre) [reply config/constants.yml:42]: Same for me, they come from googleusercontent.com so there is a CSP error :disappointed:

File: src/panel/NoResultMessage.jsx (hunk 3 above)
Comment (xem) [reply src/panel/NoResultMessage.jsx:29]: Source strings are in english, and when we surround them with `_(...)`, it allows to translate them on transifex, but that can only be done after the feature is merged on master

**Review by fatal69100 — APPROVED** [review]

**Pull request merged**